src/lib/renderer/renderer.tsx

```diff
@@ -171,7 +216,7 @@ export const useComponentType = (
 
 /**
  * Custom hook resolving the value from `ValuesContext`.
- * @external {RenderContext} Expects `RenderContext` to be available.
+ * @external {SubmissionContext} Expects `SubmissionContext` to be available.
  */
 export const useValue = (
```

Comment:
this approach is now obsolete because we don't have to fetch the value from submission.data anymore - it should be provided to formik `initialValues`, after which you can `const {values} = useFormikContext()` to obtain the form values and then `get(values, component.key)` to get the value of one particular component.

Alternatively, you can also do `const [{value}] = useField(component.key);` to get the value of the particular component from the formik state (I think this has my preference).